calico/operations/upgrading/kubernetes-upgrade.mdx

```diff
@@ -129,16 +129,15 @@ the output and then re-running the command without --dry-run to commit to the ch
 
    :::
 
-1. Use the following command to initiate a rolling update, after replacing
-   `<manifest-file-name>` with the file name of your {{version}} manifest.
+1. Use the following command to initiate a rolling update.
```

Comment:
I think it will be nicer for the user to just do
```
export version=<vx.y.z>
```

Reply:
So in stead of 
```
curl https://raw.githubusercontent.com/projectcalico/calico/v3.25.1/manifests/calico.yaml -O
curl https://raw.githubusercontent.com/projectcalico/calico/v3.25.1/manifests/canal.yaml -O
curl https://raw.githubusercontent.com/projectcalico/calico/v3.25.1/manifests/calico-policy-only.yaml -O
```

it would be:
```
curl https://raw.githubusercontent.com/projectcalico/calico/v3.25.1/manifests/calico.yaml -o upgrade.yaml
curl https://raw.githubusercontent.com/projectcalico/calico/v3.25.1/manifests/canal.yaml -o upgrade.yaml
curl https://raw.githubusercontent.com/projectcalico/calico/v3.25.1/manifests/calico-policy-only.yaml -o upgrade.yaml
```

Reply:
We can do above, but the manifests paths using {{version}} are failing in netlify (see logs). Unless someone can fix the backend?

Reply:
In the CI, the patch version is missing from URLs:

> https://raw.githubusercontent.com/projectcalico/calico/v3.26/manifests/canal.yaml

It should be v3.26.x